### PR TITLE
RA-1729: Copying 'Dispensed Medication' widget from Order Entry UI.

### PR DESCRIPTION
**Pull request by Ruhanga**
Ticket : https://issues.openmrs.org/browse/RA-1729

### CODE REVIEW

File: api/src/main/resources/messages.properties [hunk 1]
```diff
@@ -31,6 +31,7 @@ coreapps.age.days={0} Day(s)
 coreapps.patientDashBoard.editPatientIdentifier.duplicateMessage=is already in use
 coreapps.patientDashBoard.editPatientIdentifier.invalidMessage=did not pass validation
 coreapps.patientDashBoard.editPatientIdentifier.invalidFormatMessage=must be in the format
+coreapps.patientdashboard.activeDrugOrders=Dispensed Medication
```
Comment (mks-d) [inline api/src/main/resources/messages.properties]: Not sure if this should be plural.

File: omod/src/test/resources/dispensedMedication_app.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "id": "coreapps.dispensedMedication",
+  "instanceOf": "coreapps.template.dashboardWidget",
+  "description": "Show Dispenced Medication as drug orders",
```
Comment (mks-d) [inline omod/src/test/resources/dispensedMedication_app.json]:
```json
  "description": "Show dispensed medications as drug orders",
```

File: api/src/main/resources/messages.properties (hunk 1 above)
Comment (Ruhanga) [reply api/src/main/resources/messages.properties]: @mks-d, are you referring to something like `Dispensed Medications`?

File: omod/src/test/resources/dispensedMedication_app.json (hunk 2 above)
Comment (Ruhanga) [reply omod/src/test/resources/dispensedMedication_app.json]: Ok 👍

File: api/src/main/resources/messages.properties (hunk 1 above)
Comment (mks-d) [reply api/src/main/resources/messages.properties]: Yes but no ;-) This word stays singular in English. We keep it as it is.

**Pull request merged**